packages/solutions/app-tools/package.json

```diff
@@ -86,6 +86,7 @@
     "@modern-js/types": "workspace:*",
     "@modern-js/upgrade": "workspace:*",
     "@modern-js/utils": "workspace:*",
+    "@rspack/plugin-node-polyfill": "^0.1.1",
```

Comment:
`app-tools` should not depend on this package by default, maybe it can be moved to the `builder-rspack-provider`.

Reply:
In webpack-provider, it's an independent plugin which named `@modern-js/builder-plugin-node-polyfill`. how about I move it into builder-plugin-node-polyfill, and export as  builderRspackPluginNodePolyfill ?

Reply:
Why not just make the `@modern-js/builder-plugin-node-polyfill` to be compatible with Rspack?

Reply:
Rspack not support ProvidePlugin and `resolve.fallback`.  And in `@rspack/plugin-node-polyfill`, It's a limited node-polyfill support which not support global polyfill such as Buffer and process.